src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -0,0 +1,46 @@
+﻿#!/bin/bash
```

Comment:
Can you please change this to `#!/usr/bin/env bash`?  That will make the script more portable to platforms where bash is not installed directly in /bin (which will be helpful for us longer term).